src/main.sh

```diff
@@ -39,6 +42,8 @@ main() {
     log::message "Task found with the id: $task_id"
 
     export TEAMWORK_TASK_ID=$task_id
+    local -r project_id="$(teamwork::get_project_id_from_task "$task_id")"
```

Comment:
Hey @rkreutz  our ```deskclient``` builds are failing now with the error saying ```/src/main.sh: line 45: local: project_id: readonly variable```.
https://github.com/Teamwork/deskclient/pull/4845/checks?check_run_id=3280181461

Reply:
I guess removing that `-r` will fix it, will open a new PR for that

Reply:
Actually just removing the quotes should do

Reply:
It only fails for task descriptions that have more than 1 TW task link.

Reply:
Added a proposed fix for this issue here: https://github.com/Teamwork/github-sync/pull/47